playbooks/tests/tasks/network.yml

```diff
@@ -52,8 +52,8 @@
 
 # try ping internet for 5 minutes
   - name: neutron router can ping internet
-    shell: ROUTER_NS=$( ip netns show | grep qrouter- | awk '{print $1}' );
-           ip netns exec ${ROUTER_NS} ping -c 5 8.8.8.8
+    shell: DHCP_NS=$( ip netns show | grep qdhcp- | awk '{print $1}' );
+           ip netns exec ${DHCP_NS} ping -c 5 8.8.8.8
     when: neutron.l3ha.enabled is defined and neutron.l3ha.enabled|bool
```

Comment:
The `when` statement here is unnecessary here unless we have a reason to not test ping when l3ha is not enabled.